server/routes/activities/record-attendance/handlers/attendanceList.ts

```diff
@@ -90,12 +88,21 @@ export default class AttendanceListRoutes {
 
     const userMap = await this.userService.getUserMap([instance.cancelledBy], user)
 
+    if (mode === 'standalone') {
```

Comment:
The view was called from "outside" attendance journey so clear any stored session data